src/Stratis.Bitcoin/P2P/Peer/NetworkPeerFactory.cs

```diff
@@ -133,7 +133,7 @@ public NetworkPeer CreateConnectedNetworkPeer(Network network, IPEndPoint endpoi
         {
             var peerAddress = new NetworkAddress()
             {
-                Time = DateTimeOffset.UtcNow,
+                Time = this.dateTimeProvider.GetTimeOffset(),
```

Comment:
UtcNow to TimeOffset, is that intentional?

Reply:
yes, same as before, same time, different delivery. UtcNow - check DTP.GetTimeOffset how it is implemented:

```
        public virtual DateTimeOffset GetTimeOffset()
        {
            return DateTimeOffset.UtcNow;
        }

```